src/rtc/h5_prep.py

```diff
@@ -466,12 +490,24 @@ def get_metadata_dict(product_id: str,
 
     if not estimated_geometric_accuracy_bias_y:
         estimated_geometric_accuracy_bias_y = '(UNSPECIFIED)'
+    else:
+        estimated_geometric_accuracy_bias_y = float(
```

Comment:
This variable is from `cfg_in.groups.processing.geocoding.estimated_geometric_accuracy_bias_y`, and according to schema, it can be either `None` or a number. Any reason that you are casting it to `float`?

Reply:
Yes, if the user provides the `value estimated_geometric_accuracy_bias_y = 1`, the data type of the output will be integer rather than float.